app/views/components/_select_with_search.html.erb

```diff
@@ -13,7 +13,7 @@
 
 <%= content_tag :div, class: select_helper.css_classes, data: select_helper.data_attributes do %>
   <% if is_page_heading %>
-    <%= tag.h1 label_tag(id, label, class: select_helper.label_classes, id: "#{id}_label"), class: "gem-c-title__text" %>
+    <%= tag.h1 label_tag(id, label, class: select_helper.label_classes, id: "#{id}_label"), class: "gem-c-heading__text" %>
```

Comment:
It'd be nice if we didn't have a gem class outside of a component like this but I see that this makes the least amount of change. Did you consider using one of the `govuk-heading-` classes instead? It wouldn't look the same, but would reduce the assumed dependency.

Reply:
Looks like you've changed it to `govuk-heading-xl` - would `govuk-heading-m` not be closer in size? Sorry, can't see this to check.

Reply:
Ah of course, sorry got confused, ignore me.